deployments/examples/README.md

```diff
@@ -57,3 +57,10 @@ Use [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) as a wo
     <br>See [Dynamic Volume Provisioning](https://kubernetes.io/docs/concepts/storage/dynamic-provisioning/) in the kubernetes docs.
   
 _NOTE_: dynamic provisioning can interfere with the existing volume examples given.
+
+## namespace
+The namespace example uses a placement rule and special queue configuration. The pod is a simple sleep pod which will be scheduled based on the namespace it runs in. The pod does not have a queue set.
+
+* create the config volume using the local [queues.yaml](./namespace/queues.yaml) file not the standard one.
```

Comment:
suggest renaming "config volume" to "configmap", and add a link to https://github.com/cloudera/yunikorn-core/blob/master/docs/user-guide.md#create-the-configmap.